pkg/v1/tarball/image.go

```diff
@@ -162,6 +162,10 @@ func (i *image) loadTarDescriptorAndConfig() error {
 		return err
 	}
 
+	if i.manifest == nil{
```

Comment:
```suggestion
	if i.manifest == nil {
```

Reply:
Thanks. My linter didn't work...I've fixed it.